### PR TITLE
Fixing some broken links.

### PR DESCRIPTION
**Pull request by gabrieldemarmiesse**
I couldn't find the page for the examples with the newaxis function, so instead, I've put a small example.

### CODE REVIEW

File: docs/src/userguide/memoryviews.rst [hunk 1]
```diff
@@ -171,8 +171,10 @@ As for NumPy, new axes can be introduced by indexing an array with ``None`` ::
     # 2D array with shape (50, 1)
     myslice[:, None]
 
+    # 3D array with shape (1, 10, 1)
+    myslice[None, 10:-20:2, None]
+
 One may mix new axis indexing with all other forms of indexing and slicing.
-See also an example_.
```
Comment (scoder) [inline docs/src/userguide/memoryviews.rst]: The example link is at the very bottom of the file.

Comment (gabrieldemarmiesse) [reply docs/src/userguide/memoryviews.rst]: Yep. The anchor doesn't work. This scipy page does not mention `newaxis`. But I forgot to remove the link at the bottom. True.

File: docs/src/userguide/memoryviews.rst [hunk 2]
```diff
@@ -691,4 +691,3 @@ call functions in C files, see :ref:`using_c_libraries`.
 .. _new style buffers: http://docs.python.org/c-api/buffer.html
 .. _pep 3118: http://www.python.org/peps/pep-3118.html
 .. _NumPy: http://docs.scipy.org/doc/numpy/reference/arrays.ndarray.html#memory-layout
-.. _example: http://www.scipy.org/Numpy_Example_List#newaxis
```
Comment (scoder) [inline docs/src/userguide/memoryviews.rst]: I consider this page a good replacement: https://docs.scipy.org/doc/numpy/reference/arrays.indexing.html

Comment (gabrieldemarmiesse) [reply docs/src/userguide/memoryviews.rst]: Thanks for the link! I think we should still keep both the example and the link.

**Pull request merged**